iceoryx2-lang/c/iceoryx2.h

```diff
@@ -0,0 +1,6 @@
+#include <stdarg.h>
```

Comment:
I think in the long-term this header can be removed again?

Reply:
This can be removed right now. The header which is used is generated into the target directory